homeassistant/components/modbus/climate.py

```diff
@@ -21,41 +21,42 @@
 )
 import homeassistant.helpers.config_validation as cv
 
-from . import CONF_HUB, DEFAULT_HUB, DOMAIN as MODBUS_DOMAIN
+from .const import (
+    CALL_TYPE_REGISTER_HOLDING,
+    CALL_TYPE_REGISTER_INPUT,
+    CONF_CURRENT_TEMP,
+    CONF_CURRENT_TEMP_REGISTER_TYPE,
+    CONF_DATA_COUNT,
+    CONF_DATA_TYPE,
+    CONF_HUB,
+    CONF_MAX_TEMP,
+    CONF_MIN_TEMP,
+    CONF_OFFSET,
+    CONF_PRECISION,
+    CONF_SCALE,
+    CONF_STEP,
+    CONF_TARGET_TEMP,
+    CONF_UNIT,
+    DATA_TYPE_FLOAT,
+    DATA_TYPE_INT,
+    DATA_TYPE_UINT,
+    DEFAULT_HUB,
+    MODBUS_DOMAIN,
+)
 
 _LOGGER = logging.getLogger(__name__)
 
-CONF_TARGET_TEMP = "target_temp_register"
-CONF_CURRENT_TEMP = "current_temp_register"
-CONF_CURRENT_TEMP_REGISTER_TYPE = "current_temp_register_type"
-CONF_DATA_TYPE = "data_type"
-CONF_COUNT = "data_count"
-CONF_PRECISION = "precision"
-CONF_SCALE = "scale"
-CONF_OFFSET = "offset"
-CONF_UNIT = "temperature_unit"
-DATA_TYPE_INT = "int"
-DATA_TYPE_UINT = "uint"
-DATA_TYPE_FLOAT = "float"
-CONF_MAX_TEMP = "max_temp"
-CONF_MIN_TEMP = "min_temp"
-CONF_STEP = "temp_step"
-SUPPORT_FLAGS = SUPPORT_TARGET_TEMPERATURE
-HVAC_MODES = [HVAC_MODE_AUTO]
-
-DEFAULT_REGISTER_TYPE_HOLDING = "holding"
-DEFAULT_REGISTER_TYPE_INPUT = "input"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_CURRENT_TEMP): cv.positive_int,
         vol.Required(CONF_NAME): cv.string,
         vol.Required(CONF_SLAVE): cv.positive_int,
         vol.Required(CONF_TARGET_TEMP): cv.positive_int,
-        vol.Optional(CONF_COUNT, default=2): cv.positive_int,
```

Comment:
If we are renaming a config option, we need to label this as a breaking change so users know they need to update their configuration (or revert the name change)

Reply:
We are renaming the constant, not the content of the constant, a slight difference. Seen from a use POW, there are no changes. (but a very good catch).

Reply:
Got it 

Old
```
const.py:CONF_DATA_COUNT = "data_count"
```
New
```
climate.py:CONF_COUNT = "data_count"
```